### DISCUSSION TITLE
When reusing same tween, it's not starting from current value

### DISCUSSION THREAD

**Discussion by optimus007** [tweenjs/tween.js#635] [Q&A] [answered]
I have a vector3() called focusPoint which needs to tween to the cursor objects's position on every click

i create a tween 

```
  const focusTween = new Tween(focusPoint)
    .duration(1500)
    .easing(Easing.Elastic.Out)
```

then on pointer events  i use

```
...
 focusTween.to({
      x: cursor.position.x,
      y: cursor.position.y,
      z: cursor.position.z,
    })
    if (focusTween.isPlaying()) {
      focusTween.stop()
    }
    focusTween.start()
```

this works for the first click then second click onwards it does not start from where it was before (jumps to end frame)

anyway to overcome this ?

**Comment by trusktr** [comment, marked as answer]
Hello! There is a new `tween.startFromCurrentValues()` method, which is an alias for `tween.start(undefined, true)`. This will start the tween again from the current values instead of from the beginning. Needs doc update.
